tests/tools/oesign-engine/test-engine/oesign-test-engine.c

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+//
+// oesign-test-engine: minimal engine with predictable output to test oesign engine support
+
+#include <openssl/engine.h>
+#include <openssl/types.h>
+#include <openssl/bio.h>
+#include <openssl/pem.h>
+#include <sys/stat.h>
+
+static const char *test_engine_id   = "oesign-test-engine";
+static const char *test_engine_name = "oesign test engine";
+
+#if 0
```

Comment:
why is there an #if 0?
Recommend deleting code if it's never used.

Reply:
done